lib/compile.js

```diff
@@ -14,19 +14,27 @@ module.exports = {
       .fromCallback(cb => compiler.run(cb))
       .then(stats => {
 
-        if (!this.multiCompile) {
-          stats = { stats: [stats] };
-        }
-
-        const compileOutputPaths = [];
-        const consoleStats = this.webpackConfig.stats || {
```

Comment:
Wouldn't just
```
const consoleStats = this.webpackConfig.stats || _.get(this, 'webpackConfig[0].stats') || { ...
```
be enough instead of the ifs below? Then it would try plain, then array and finally fall back to the defaults.